README.md

```diff
@@ -10,6 +10,9 @@ pranjul139
 ```
 npm install
 ```
+```
+npm install --save vue-animejs
```

Comment:
This is not required, it is automatically added in the package.json as a dependency